cmd/snap-confine/mount-support.c

```diff
@@ -638,6 +652,7 @@ void sc_populate_mount_ns(struct sc_apparmor *apparmor, int snap_update_ns_fd,
 			{"/media", true},	// access to the users removable devices
 #endif				// MERGED_USR
 			{"/run/netns", true},	// access to the 'ip netns' network namespaces
+			{"/mnt",.is_optional = true},	// to support the removable-media interface
```

Comment:
Can you add a space before '.is_optional'?

That said, why is '/mnt' optional? The FHS says it is required: http://www.pathname.com/fhs/pub/fhs-2.3.html#REQUIREMENTS so I question "our desire for having almost-empty bases
to be functional". Not a blocker, but please discuss and reaffirm that base snaps are allowed to violate the FHS.

Reply:
I did but `make fmt` apparently removed it.

This is a great question: it is optional because as a part of the base snap work we said we'd be happy with mostly empty bases. There's an existing test that doesn't contain `/mnt` and passes. If this is required that test would fail. I was not feeling comfortable with a fix that involves changes to existing bases as in theory someone may have a private base that I cannot adjust.